katas/scrabble-score/test.js

```diff
@@ -0,0 +1,70 @@
+function scrabbleScore(word) {
```

Comment:
if using TDD, it would be good if we dont have the tests other than the basic 1 test to get going. 
If you need to keep these tests, may be we could rename the file as sample_test.js and have another file called test.js